lessons/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -0,0 +1,116 @@
+
+@lesson-prereqs{absolute-value-and-opposites-more}
+
+@keywords{exponent, base}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Translate between words, mathematical expressions, and Circles of Evaluation.
+* Evaluate expressions involving exponents and one other operation.
+* Evaluate expressions involving exponents and negations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyze the structure of expressions with exponents.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* Circles of Evaluation will not help students to visualize or understand the meaning of exponents as repeated multiplication. Rather, they will help students parse expressions that involve exponents. We recommend ensuring that students are comfortable evaluating exponents before jumping into this lesson.
+
+|===
+
+== Expressions with Exponents and One Operation @duration{30 minutes}
+
+=== Overview
+
+Sudents use Circles, words, and mathematical notation to analyze the structure of expressions with exponents and one operation.
+
+=== Launch
+
+[.lesson-instruction]
+- Show students the following expression: @math{2^3}
+- One way of reading this expression is "two raised to the power of three." Do you know any other ways of reading this expression with an exponent?
+** _Two to the third power; two to the third; two cubed._
+- What does it mean to _cube_ a number?
+** We multiply it by itself three times.
+- What does this expression evaluate to?
+** @math{2^3 = 2 \times 2 \times 2 = 8}
+
+Clarify and reinforce some key concepts about exponents via class discussion:
+
+- The number on the left is called the _base_; that number is multiplied by itself when we apply the exponent.
+
+- The number next to the base is called the _exponent_; it indicates how many times to multiply the base.
+
+- "Cubing" is the same as "raising to the third power", and "squaring" is the same as raising to the second power.
+
+- There is no special terminology for any other exponents. (The subsequent lesson will address why two and three are referred to as "squaring" and "cubing.")
+
+Explain to students that we can represent expressions with exponents as Circles of Evaluation. The expression we used is called `num-expt`; when we use `num-expt`, the value on the left is the _base_, and the value on the right is the exponent. The Circle of Evaluation for @math{2^3} looks like this:
+
+[.centered-image]
+@show{(coe '(num-expt 2 3))}
+@clear
+
+Now, have students analyze the two Circles below.
+
+[.embedded, cols="^.^1,^.^1", grid="none", stripes="none" frame="none"]
+|===
+|@math{5^2}							| @math{2^5}
+|@show{(coe  '(num-expt 5 2))}		| @show{(coe  '(num-expt 2 5))}
+|===
+
+[.lesson-instruction]
+- Can we switch the order of the exponent and the base, as we could when adding or multiplying values?
```

Comment:
You spent all this time building up the term "equivalence", finally introduced it a few lessons ago, and now it's gone! Let's _use it_:  "is it TRUE or FALSE that the following two expressions are equivalent? Why?"